### PR TITLE
add 'migrating from avado'

### PR DESCRIPTION
**Pull request by Hydepwns**
lfg #csmtestnet

### CODE REVIEW

File: docs/user/videos-and-tutorials/guides/migrating-from-avado.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+(**4**) Once you've logged in, complete the installation through the terminal as documented [here.](./migrating-from-avado.md#complete-installation)
+
+### Complete Installation 
+
+These are additional steps to complete the installation of Dappnode software in case you cannot find the Dappnode wifi after following the [above instructions](./migrating-from-avado.md#how-to-migrate-your-avado-node-to-a-dappnode).
+
+Follow these steps after having logged into your now semi-transitioned Avado device. You should see the following screen:
+
+![Dappnode Screen](/img/Dappnode_screen.png)
```
Comment (Marketen) [inline docs/user/videos-and-tutorials/guides/migrating-from-avado.md:148]: This image wasn't added in `/static/img`

`Error: Image static/img/Dappnode_screen.png used in docs/user/videos-and-tutorials/guides/migrating-from-avado.md not found.`

**Review by Marketen — CHANGES_REQUESTED** [review]

File: docs/user/videos-and-tutorials/guides/migrating-from-avado.md (hunk 1 above)
Comment (Hydepwns) [reply docs/user/videos-and-tutorials/guides/migrating-from-avado.md:148]: fixing now

Comment (Hydepwns) [reply docs/user/videos-and-tutorials/guides/migrating-from-avado.md:148]: @Marketen hopefully should be good now! https://github.com/dappnode/DAppNodeDocs/pull/423/commits/01307867c931f2779dd45ab865176376bdfc9bf6

Comment (Marketen) [reply docs/user/videos-and-tutorials/guides/migrating-from-avado.md:148]: Seems like it! Thanks for this great guide

File: docs/user/videos-and-tutorials/guides/migrating-from-avado.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+
+:::caution Warning
+**Important:** Before following this guide to install DappnodeOS on your Avado device, make sure to first backup any data if you are running a validator or any other dApp.
+**The migration process will otherwise wipe all existing data.**
+
+:::
+
+## Introduction
+
+This guide was adapted from the original [HOPR Migration Guide](https://github.com/hoprnet/hopr-docs/blob/d8d1ec05e5938eb29254188b711eec47a71a92ab/versioned_docs/version-v2.1.3/node/using-avado.md) & will walk you through the process of migrating your Avado node to a Dappnode.
```
Comment (Marketen) [inline docs/user/videos-and-tutorials/guides/migrating-from-avado.md:11]: Hey @Hydepwns! What do you think about adding the text in bold here? 

This guide was adapted from the original HOPR Migration Guide & will walk you through the process of migrating your Avado node to a Dappnode **, due to the discontinuation of support for HOPR from AVADO**

I can merge it after this is added, or add it myself later if you prefer.

**Review by Marketen — CHANGES_REQUESTED** [review]

File: docs/user/videos-and-tutorials/guides/migrating-from-avado.md (hunk 2 above)
Comment (Hydepwns) [reply docs/user/videos-and-tutorials/guides/migrating-from-avado.md:11]: adding this now sir

Comment (Hydepwns) [reply docs/user/videos-and-tutorials/guides/migrating-from-avado.md]: Semantic update to:
"... & will walk you through the process of migrating your Avado hardware to DappnodeOS, due to the discontinuation of support for HOPR from AVADO."

**Review by Hydepwns — COMMENTED** [review]
Semantic update completed

**Comment by Marketen** [comment]
Merged! Sorry for the wait and thanks for this awesome guide

**Pull request merged**